pkgs/applications/graphics/geeqie/default.nix

```diff
@@ -12,31 +14,26 @@ stdenv.mkDerivation rec {
     owner = "BestImageViewer";
     repo = "geeqie";
     rev = "v${version}";
-    sha256 = "sha256-O+yz/uNxueR+naEJG8EZ+k/JutRjJ5wwbB9DYb8YNLw=";
+    sha256 = "sha256-0GOX77vZ4KZkvwnR1vlv52tlbR+ciwl3ycxbOIcDOqU=";
   };
 
-  patches = [
-    # Do not build the changelog as this requires markdown.
-    (fetchpatch {
-      name = "geeqie-1.4-goodbye-changelog.patch";
-      url = "https://src.fedoraproject.org/rpms/geeqie/raw/132fb04a1a5e74ddb333d2474f7edb9a39dc8d27/f/geeqie-1.4-goodbye-changelog.patch";
-      sha256 = "00a35dds44kjjdqsbbfk0x9y82jspvsbpm2makcm1ivzlhjjgszn";
-    })
-  ];
-
   postPatch = ''
     patchShebangs .
+    # libtiff detection is broken and looks for liblibtiff...
+    # fixed upstream, to remove for 2.1
+    substituteInPlace meson.build --replace 'libtiff' 'tiff'
   '';
 
-  preConfigure = "./autogen.sh";
-
   nativeBuildInputs =
-    [ pkg-config autoconf automake gettext intltool
+    [ pkg-config gettext intltool
       wrapGAppsHook doxygen
+      meson ninja xxd
     ];
 
   buildInputs = [
     gtk3 lcms2 exiv2 libchamplain clutter-gtk ffmpegthumbnailer fbida
+    libarchive djvulibre libheif openjpeg libjxl libraw lua5_3 poppler
+    gspell libtiff libwebp
   ];
 
   postInstall = ''
```

Comment:
This sed below might also need adding more paths; it calls e.g. mogrify (imagemagick) which isn't added to PATH anywhere (it's not part of the build deps currently, so we can't add it without adding yet another dep...)

But that was already the case in 1.7, so this isn't a new problem -- I'll leave that to whoever cares.